articles/protocols/saml/saml-apps/atlassian.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Learn how to configure Single Sign-on (SSO) with Atlassian.
+topics:
+    - saml
+    - identity-providers
+    - atlassian
+contentType:
+  - how-to
+useCase:
+  - add-idp
+---
+
+# Atlassian SAML Configuration
+
+You can configure Auth0 as the SAML SSO identity provider for Atlassian.
+
+## Prerequisites
+
+* A verified domain in your [Atlassian Organization](https://admin.atlassian.com/).
+* [Atlassian Access](https://www.atlassian.com/software/access)
+
+## Configure Auth0 identity provider application
+
+1. Go to [Dashboard > Applcations](${manage_url}/#/applications) and click **Regular Web Applications** to create an app.
+2. Add connections to your app. Make sure that the correct set of Auth0 connections are associated with your identity provider app.
+3. Go to **Addons** tab and enable the **SAML2 Web App** addon. 
+4. Click the **Usage** tab. Here you will find information to populate SAML metadata in Atlassian Access. Note the settings. 
+
+## Configure Atlassian SAML connection
+
+1. Log in to your [Atlassian Admin](https://admin.atlassian.com/) dashboard.
+2. Go to **Organization > Security > SAML single sign-on**.
+3. Copy the configuration settings from the Auth0 SAML2 Web App addon usage page:
+
+| Setting | Description|
+| -- | -- |
+| **Identity provider Entity ID** | `urn.TENANT.auth0.com` |
+| **Identity provider SSO URL** | `https://TENANT.auth0.com/samlp/APP_ID` |
+| **Public x509 certificate** | Content from your .pem file: `https://TENANT.auth0.com/pem` |
+
+4. Click **Save configuration**.
+5. On the summary page, copy the **SP Assertion Consumer Service URL**.
```

Comment:
It would be good to attach a screenshot of the completed setup here. e.g. https://raw.githubusercontent.com/abbaspour/dev.to/master/blog-posts/2020-04-14-atlassian/assets/07-idp-url.png

Reply:
We have a policy of not including screen shots from other company's sites.